.eslintrc

```diff
@@ -0,0 +1,158 @@
+rules:
+  indent:
+    - 2
+    - 2
+  quotes:
+    - 2
+    - single
+  linebreak-style:
+    - 2
+    - unix
+  semi:
+    - 2
+    - never
+  array-bracket-spacing: 
+    - 2
+    - "never"
+  block-scoped-var: 2
+  brace-style: 
+    - 2
+    - "stroustrup"
+    - 
+      allowSingleLine: true
+  comma-dangle: 
+    - 2
+    - "never"
```

Comment:
The dangling comma? Yeah - me too, my preferred standard is actually *always*.  That said… it's not valid in some browsers which made me think we should allow it never.

That said I wanted to ping you and @tortillaj and see what you guys think aobut [isaacs's strongly preferred standard](https://gist.github.com/isaacs/357981).  I find his argument (after reading the text, not just looking at the code) very compelling.  It looks gross to me but I'm sure I'd get used to it and he's right - it makes errors exceedingly obvious.